clang/lib/DPCT/ApiMapping/ApiMapping.inc

```diff
@@ -0,0 +1,31 @@
+//===--------------- ApiMapping.inc ---------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+REGISTER_ENTRY("cudaStreamSynchronize", "sycl::queue::wait",
+"CUDA API\n"
```

Comment:
will this file be auto generated by scripts from doc?

Reply:
Yes, In feature it will be.